Dockerfile

```
@@ -0,0 +1,33 @@
+FROM ubuntu:22.04
```

Гораздо эффективнее будет взять контейнер сразу с нодой и тогда можно будет удалить практически половину строк кода здесь. 
```suggestion
FROM node:16
```